src/conv-ldb/cldb.neighbor.C

```diff
@@ -391,6 +391,22 @@ void CldEnqueueGroup(CmiGroup grp, void *msg, int infofn)
   CmiSyncMulticastAndFree(grp, len, msg);
 }
 
+void CldEnqueueWithinNode(void *msg, int infofn)
+{
+  int len, queueing, priobits,i; unsigned int *prioptr, start, size;
```

Comment:
Yes these files are kind of a nightmare. From what I've seen there is very little difference. I personally am not even sure how the different versions get used, but I would hope that is at least documented somewhere. For such a small, probably rarely used feature, these files add an annoying amount of development overhead.

Reply:
Rather than making a new file for this commit, I think I will follow up with a cleanup of either these files, or the broadcast code in general. Or both.

Reply:
Yeah, that's a good idea. We can do this cleanup in a separate cleanup issue.